cypress/integration/26-register_registration/creator_navigates_to_resource_type_tab/test_invalid_values.js

```diff
@@ -0,0 +1,43 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+And('they enter an invalid value in fields:', (dataTable) => {
+  cy.get('@fields').then((fields) => {
+    if (fields !== {}) {
+      dataTable.rawTable.forEach((field) => {
+        cy.get(`[data-testid=${fields[field[0]]}]`).type('invalid');
+      });
+    }
+  });
+});
+
+const testMandatoryFields = (dataTable) => {};
+
+Then('they can see "Mandatory" error messages for fields:', (dataTable) => {
+  cy.get('@fields').then((fields) => {
+    if (Object.keys(fields).length > 0) {
+      dataTable.rawTable.forEach((value) => {
+        cy.get(`[data-testid=${fields[value[0]]}]`).within((field) => {
+          cy.wrap(field).contains('is required');

```

Comment:
alle feilmeldinger skal ha `class=MuiError`, så kan sjekke på det istedet for tekst